packages/consumption/src/consumption/ConsumptionController.ts

```diff
@@ -64,6 +65,11 @@ export class ConsumptionController {
         return this._drafts;
     }
 
+    private _tags: TagsController;
+    public get tags(): TagsController {
+        return this._tags;
+    }
```

Comment:
we had the decision to NOT have tags in an own controller b/c it would have to be named AttributeTags which is ugly. Please move that logic back to AttributesController.

Reply:
Yea but with the decision that files can have tags we will also need those methods for files also the AttributeControler is already very bloated.

Reply:
> files can have tags

as soon as this is not tackled by the backbone, I don't think this has to be considered when implementing this feature.

> the AttributeControler is already very bloated

moving stuff to a completely new module is not the only solution, so no argument